src/textual/_wait.py

```diff
@@ -2,7 +2,7 @@
 from time import monotonic, process_time
 
 SLEEP_GRANULARITY: float = 1 / 50
-SLEEP_IDLE: float = SLEEP_GRANULARITY / 2.0
+SLEEP_IDLE: float = SLEEP_GRANULARITY / 10.0
```

Comment:
This will make `wait_for_idle` potentially sleep a little more, and hopefully reduce the flaky tests even further.